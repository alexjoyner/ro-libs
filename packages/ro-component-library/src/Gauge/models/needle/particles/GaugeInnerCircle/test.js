import React from 'react';
import { shallow } from 'enzyme';
import { GaugeInnerCircle } from '.';

describe('GaugeInnerCircle component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<GaugeInnerCircle />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
