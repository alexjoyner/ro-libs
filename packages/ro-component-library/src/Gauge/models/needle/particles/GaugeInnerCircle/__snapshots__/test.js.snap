// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GaugeInnerCircle component Should match snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "GaugeInnerCircle-sc-75fsjf-0",
        "isStatic": true,
        "rules": Array [
          "stroke:#34495e;stroke-width:1;stroke-dasharray:314.160;stroke-dashoffset:314.160;",
        ],
      },
      "displayName": "GaugeInnerCircle",
      "render": [Function],
      "styledComponentId": "GaugeInnerCircle-sc-75fsjf-0",
      "target": "circle",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
